lib/y2logsstep.pm

```diff
@@ -85,6 +85,8 @@ sub workaround_dependency_issues {
         while (check_screen('dependency-issue', 5)) {
             wait_screen_change { send_key 'alt-1' };
             select_conflict_resolution;
+            # Refer ticket https://progress.opensuse.org/issues/48266
+            wait_still_screen;
```

Comment:
The default stilltime is 7 seconds which will cause big delays here. Can you please try if maybe `wait_still_screen(1)` is enough? At best instead of `wait_still_screen` have a proper `assert_screen` with the screen we are actually waiting for

Reply:
At first, I want do workaround to fix this problem. But I found a better way to fix this instead of workaround, but I didn't change the PR subject, So maybe this confuse you.
As the code while (check_screen('dependency-issue', 5)),  'dependency-issue' needles always show up even change to next screen, so I add 'wait_still_screen' to confirm the screen has changed before next loop. Maybe I can reduce the wait time to 2 instead of using the default time.

Reply:
Because at the loop, it may change screen ten times, during this, we should wait screen not change, then go to next loop. If use assert_screen, it will compare the screen every time. I think it is not a better way.